Sources/SwiftDocC/Infrastructure/DocumentationContext.swift

```diff
@@ -155,6 +155,10 @@ public class DocumentationContext: DocumentationContextDataProviderDelegate {
     var assetManagers = [BundleIdentifier: DataAssetManager]()
     /// A list of non-topic links that can be resolved.
     var nodeAnchorSections = [ResolvedTopicReference: AnchorSection]()
+    /// A list of reference aliases.
+    ///
+    /// When multiple references resolve to the same documentation, use this to find the main reference that is associated with the node in the topic graph.
+    var referenceAliases = [ResolvedTopicReference: ResolvedTopicReference]()
```

Comment:
~Could a reference have multiple aliases? For the Swift+Objective-C case no, but for APIs that are available in more than 2 languages, it's possible. I'll leave it up to you on whether we should model that now, or leave it for a future PR.~

Reading this more, I realise that this actually a mapping from the alias to the canonical reference, rather than the inverse. So this does support multiple aliases for the same symbol. I think we should document what the key/values are.

Reply:
Updated